packages/remix/src/utils/vendor/response.ts

```diff
@@ -153,11 +156,5 @@ export function isDeferredData(value: any): value is DeferredData {
 export function isRouteErrorResponse(value: any): value is ErrorResponse {
   const error: ErrorResponse = value;
 
-  return (
-    error != null &&
-    typeof error.status === 'number' &&
-    typeof error.statusText === 'string' &&
-    typeof error.internal === 'boolean' &&
```

Comment:
Looks like `error.internal` does not exist in the latest types

Reply:
should we keep it around for backwards compatibility?

Reply:
Turns out this function is already exported. So I removed it and `isDeferredData` to use the `peerDependency`. 
https://github.com/getsentry/sentry-javascript/pull/16218/commits/7ef5353399a542fab535507ac7ccd69f67ad0efd